.github/workflows/nix.yml

```diff
@@ -384,6 +384,7 @@ jobs:
         run: |
           echo "${CONAN_GLOBAL_CONF}" >> $(conan config home)/global.conf
           conan config install conan/profiles/ -tf $(conan config home)/profiles/
+          conan export --version 2.3.1 external/wamr
```

Comment:
move it to action.yml

Reply:
How then it  export soci/snapi/rocksdb?

Reply:
I'm not sure. I did a ctrl-f in all the yaml files to see where else it might be exported and I couldn't figure it out. I figured we can work with the core ledger team during code review to clean all this up more.

Reply:
But we can figure out it here, anyway this merge is not urgent so it can wait

Reply:
I'd prefer to fix it in a future PR, since it's dependent on other people's availability